src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -86,15 +96,21 @@ internal class ServicePageGenerator
                     langs = lang?.ToObject<List<string?>>();
                 }
 
-                var pageType = node.LandingPageType.Value;
                 results.Add(servicePagePath);
                 var servicePageToken = new ServicePageModel(name, fullName, href, uid, children, langs, pageType);
                 _input.AddGeneratedContent(servicePagePath, JsonUtility.ToJObject(servicePageToken), "ReferenceContainer");
             }
 
             foreach (var item in node.Items)
             {
-                GenerateServicePageFromTopLevelTOC(item, results, $"{directoryName}/{filename}");
+                if (!string.IsNullOrEmpty(node.LandingPageType.ToString()) && node.LandingPageType == LandingPageType.Root)
+                {
+                    GenerateServicePageFromTopLevelTOC(item, results, $"{directoryName}");
+                }
+                else
```

Comment:
if `node.LandingPageType` is empty, shall we generate service page for it?

Reply:
Although `node.LandingPageType` may be empty, it's sub-sub items may have `LandingPageType`.
Although I haven't found cases like this, I think it deserves to be considered.........

Reply:
Then we can just remove `!string.IsNullOrEmpty(node.LandingPageType.ToString())` here

Reply:
![image](https://user-images.githubusercontent.com/67643974/91700640-cc217800-eba8-11ea-842e-ac17cb67b2ed.png)

Reply:
Fixed.